templates/web/base/reports/_other_districts-list.html

```diff
@@ -0,0 +1,58 @@
+[% IF covered.size %]
```

Comment:
I think this should be `parish_ward.size`? There are a couple of other mentions of `covered` in this file still.

Reply:
Realised that I'd broken the main bodies links, so fix for that as well.